joomla/joomla-cms
# [4.0] Deprecate unused $iconOver

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30211

### Summary of Changes
Removes unused $iconOver param.
In 1.5 days this was used as an override ( not hover as stated now ) but has never been implemented in 4.+ so might as well just remove it.  If its determined later that its useful we can always create it because as it is now it does NOTHING.

### Testing Instructions
notice that toolbar icons display.
apply pr.
make sure toolbar icons still display.
hover over icons notice the icons remain.

### Actual result BEFORE applying this Pull Request
nothing noted

### Expected result AFTER applying this Pull Request
no visual changes.


### Documentation Changes Required
Deprecate $iconOver in 5.x
